packages/grafana-e2e/src/flows/login.ts

```diff
@@ -1,6 +1,6 @@
 import { e2e } from '../index';
 
-export const login = (username: string, password: string) => {
+export const login = (username: string = 'admin', password: string = 'admin') => {
   e2e().logToConsole('Trying to login with:', { username, password });
```

Comment:
should we obfusticate the password?  maybe show just the first few characters?

Reply:
I was questioning this log entry as well. Is there a point in logging the values if we obfuscate them?

Reply:
We could do:
* Trying to log in
* Trying to login with: admin [hidden]